tests/vmi_multus_test.go

```diff
@@ -834,15 +836,15 @@ var _ = Describe("[Serial]SRIOV", func() {
 			// it's hard to match them.
 		})
 
-		// pingThroughSriov instantiates two VMs connected through SR-IOV and
-		// pings between them.
+		// createSriovVMs instantiates two VMs connected through SR-IOV.
 		// Note: test case assumes interconnectivity between SR-IOV
 		// interfaces. It can be achieved either by configuring the external switch
 		// properly, or via in-PF switching for VFs (works for some NIC models)
-		pingThroughSriov := func(cidrA, cidrB string) {
+		createSriovVMs := func(cidrA, cidrB string) (*v1.VirtualMachineInstance, *v1.VirtualMachineInstance) {
```

Comment:
The refactoring looks nice to me, but I would use a separate commit here as it is hard to compute if there is no change in logic. Please do this refactoring as a pre-commit to the new assertion.

Another general note: I think that it is more appropriate to introduce the test coverage with the change in the same commit, as they are closely related. When reviewing the code per commit, it will be much easier to confirm that it tests the change in the production code. This is true for unit and e2e tests alike.
This is not a hard rule, as in some cases the change is too large and such a separation is unavoidable. But this one is pretty simple.
(it also has the advantage of not needing to explain the test again in the commit message)